docs/concepts/fundamentals/component_views_urls.md

```diff
@@ -0,0 +1,128 @@
+_New in version 0.34_
```

Comment:
I renamed the old `components_as_views` to `component_views_urls.md`, but git didn't pick it up right. But I've updated links, the section on using components as views, and then added section on getting component URLs.